packages/react-components/src/components/Alert.spec.tsx

```diff
@@ -0,0 +1,86 @@
+import * as React from 'react';
+import { render, fireEvent } from '../test-utils';
+import { Alert, AlertSize, AlertType } from './Alert';
+
+const customClass = 'my-css-class';
+const bannerContent = 'This example content';
+const mockedFunction = jest.fn();
+
+describe('<Banner> component', () => {
```

Comment:
I think we test here `Alert` instead of `Banner` ;) but correct me if I am wrong.